### PR TITLE
Reimagine 2 devpost rebased

### PR DESCRIPTION
**Pull request by MGerrior**
https://trello.com/c/OxCBDUTw

### CODE REVIEW

File: app/views/reimagine2/devpost/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<footer id="devpost-footer">
+  <div class="row">
+    <div class="large-8 small-9 columns">
+      <nav id="site-footer-primary-nav">
+        <ul class="">
+          <li><a href="http://post.devpost.com/">Devpost for Organizations</a></li>
+          <li><a href="http://info.devpost.com/about">About</a></li>
+          <li><a href="http://info.devpost.com/jobs">Careers</a></li>
+          <li><a href="http://blog.challengepost.com/">Blog</a></li>
```
Comment (rossta) [inline app/views/reimagine2/devpost/_footer.html.erb]: Let's change this to `blog.devpost.com`

File: app/views/reimagine2/devpost/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          <li><a href="http://info.devpost.com/contact">Contact</a></li>
+          <li><a href="https://challengepost.desk.com/">Help</a></li>
```
Comment (rossta) [inline app/views/reimagine2/devpost/_footer.html.erb]: And this to http://help.devpost.com

**Comment by rossta** [comment]
Thanks for tackling this. Just a couple tweaks. Also, we'll want to bump the platform and marketing apps as well, yes?

File: app/views/reimagine2/devpost/_footer.html.erb (hunk 1 above)
Comment (MGerrior) [inline app/views/reimagine2/devpost/_footer.html.erb]: Good call, updated.

File: app/views/reimagine2/devpost/_footer.html.erb (hunk 2 above)
Comment (MGerrior) [inline app/views/reimagine2/devpost/_footer.html.erb]: Updated, thanks.

**Comment by MGerrior** [comment]
Thanks @rossta, yes we need to bump platform and marketing as well. My idea was to merge this into master first since none of the other apps point to master, and then I would open PRs for both of those apps pointing to master after that instead of having them point to this, then merge this, then update them to point to master, etc...

**Pull request merged**